frontend/src/components/DashboardDescriptionListGroup.tsx

```diff
@@ -110,7 +109,7 @@ const DashboardDescriptionListGroup: React.FC<DashboardDescriptionListGroupProps
           </Flex>
         </DescriptionListTerm>
       )}
-      <DescriptionListDescription className={isEmpty && !isEditing ? text.disabledColor_100 : ''}>
+      <DescriptionListDescription className={isEmpty && !isEditing ? text.textColorDisabled : ''}>
```

Comment:
Note: you will need to conditionally render `aria-disabled` to get passed the accessbility testing. See https://github.com/kubeflow/model-registry/pull/434